doc/nrf/gs_installing.rst

```diff
@@ -467,6 +467,14 @@ To install |SES| Nordic Edition, complete the following steps:
     #. Copy-paste the license key and click :guilabel:`Install License`.
        The license activation window will close and SES will open the Project Explorer window.
 
+.. note::
+    Notifications about newer versions of SES are disabled in SES Nordic Edition.
+    This is because of the custom |NCS| options available only in the Nordic Edition.
+    You won't be able to use SES with |NCS| if you install the latest standard version of SES.
```

Comment:
```suggestion
    This is because of the custom |NCS| options are available only in the Nordic Edition.
    You will not be able to use SES with |NCS| if you install the latest standard version of SES.
```